docs/operator/configuration_parameters.md

```diff
@@ -522,46 +522,83 @@ a setting already specified in the Configuration File.
-- **naming_convention**: <!--??--> Default: `None`.
-- **physgroup**: <!--??--> Default: ` `.
-- **quota_approvers**: <!--??--> Separated by commas. Default: `None`.
-- **restricted_read** <!--??--> Boolean. If True, only allow transfers
-  from this RSE if started by an account with admin privileges.
-  Default: `False`
-- **restricted_write** <!--??--> Boolean. Same as `restricted_read`, but for
-  transfers towards this RSE. Default: `False`
-- **rule_approvers**: <!--??--> Separated by commas. No default.
-- **rule_deleters**: <!--??--> Separated by commas. No default.
-- **site** <!--??-->
-- **skip_upload_stat**: <!--??--> Boolean. Default: `False`.
-- **source_for_total_space**: <!--??--> Default: `storage`.
-- **source_for_used_space**: <!--??--> Default: `storage`.
-- **staging_buffer** <!--??-->
+- **associated_sites**: String. Separated by commas. Used for chaining of subscriptions so that transfers to one RSE will also be mirrored to associated_sites. No default.
+- **archive_timeout**: Integer: Only used for transfers with a tape destination. Controls the number of seconds the FTS3 transfer manager will wait for the tape archival of the file to go `FAILED` or `FINISHED`. No default.
+- **auto_approve_bytes**: String: Upper limit for the size in bytes of a DID for which rules will be automatically approved. Example: `500GB`. No default.
+- **auto_approve_files**: Integer: Upper limit for the number of files covered by a rule which will be automatically approved. No default.
+- **available_for_multihop**:  Boolean. If True, allow to use this RSE as an intermediate hop in a multi-hop transfer. Default: `False`.
+- **block_manual_approval**: Boolean. Disable manual rule approval for this RSE. Default: `False`.
+- **bittorrent_tracker_addr**: String. Used to configure the URL of the bittorrent tracker API when using the torrent transfer manager. No Default.
+- **checksum_key**: String. Used to specify an alternate RSE attribute to search for supported checksums beyond those with global support (ADLER32, MD5). Default: `supported_checksums`.
+- **country**: String. Used for country level granularity of RSE selectors. No default.
+- **decommission**: Boolean. Indicates to the RSE Decommissioning Daemon that this RSE is to be decommissioned. Default: `False`.
+- **driver_name_rse_attribute**: String. Used to specify alternate drivers when using the Bittorrent transfer manager. Default: `bittorrent_driver`.
+- **fts**: String. Specify the REST API URL of the FTS3 transfer manager. No default.
+- **greedyDeletion**: Boolean. Allow files without a rule locking them to be deleted by a Reaper Daemon. Default behavior only marks a file for deletion when there is no space on an RSE for a new required file. Default: `False`.
+- **group_by_rse_attribute**: String. Control the RSE attribute (such as `country`) which transfer source RSEs will be grouped by when determining an appopriate transfer source. Default: `UNKNOWN`.
```

Comment:
Is country an attribute? I thought it was a setting?

Reply:
`country` is an attribute.
`country_name` is a setting.

Lends some credence to the statement this morning that the RSE settings and attributes could use some cleanup.

Reply:
I don't think that the `country_name` setting does anything anymore, in lieu of the  `country` attribute, so I left it out.